tests/catch/unit/stream/hipMultiStream.cc

```diff
@@ -49,13 +49,16 @@ TEST_CASE("Unit_hipMultiStream_sameDevice") {
     HIP_CHECK(hipStreamCreate(&streams[i]));
     HIP_CHECK(hipMalloc(&data[i], NN * sizeof(float)));
     hipLaunchKernelGGL(kernel, dim3(1), dim3(1), 0, streams[i], data[i], xd, NN);
+    HIP_CHECK(hipGetLastError());
     hipLaunchKernelGGL(HIP_KERNEL_NAME(nKernel), dim3(1), dim3(1), 0, 0, yd);
+    HIP_CHECK(hipGetLastError());
     HIP_CHECK(hipFree(data[i]));
     HIP_CHECK(hipStreamDestroy(streams[i]));
   }
   HIP_CHECK(hipMemcpy(&x, xd, sizeof(float), hipMemcpyDeviceToHost));
   HIP_CHECK(hipMemcpy(&y, yd, sizeof(float), hipMemcpyDeviceToHost));
   REQUIRE(x == Approx(y));
+  for (auto& stream : streams) HIP_CHECK(hipStreamDestroy(stream));
```

Comment:
Aren't we already destroying the streams inside the for loop above? We would get an error for invalid stream on both AMD and Nvidia platform here causing the test to fail.

Reply:
Made a mistake during merge